hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -101,19 +144,56 @@ def lambda_handler(event, context):
 
 
 # TODO move to utils library
-def check_db_for_matches():
+def gen_matches() -> t.List:
+    table = dynamodb.Table(DYNAMODB_TABLE)
+    records = PDQMatchRecord.get_from_time_range(table)
+    results = [
+        {
+            "content_id": record.content_id,
+            "signal_id": record.signal_id,
+            "signal_source": record.signal_source,
+            "updated_at": record.updated_at.isoformat(),
+            "reactions": "TODO",
+        }
+        for record in records
+    ]
+    return results
```

Comment:
Nit: you don't have to use the results variable, you can return the comprehension directly